packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -29,6 +29,13 @@ export interface IOverflowListProps<T> extends IProps {
      */
     items: T[];
 
+    /**
+     * The number of visible items will never be lower than the number passed to
+     * this prop.
+     * @default 0
+     */
+    minVisibleItems?: number;
```

Comment:
test please ⚖️

Reply:
perhaps the same treatment as #2622 is necessary here (adding to `shouldComponentUpdate`)?